webapi/src/main/scala/org/knora/webapi/store/triplestore/http/HttpTriplestoreConnector.scala

```diff
@@ -607,6 +600,65 @@ class HttpTriplestoreConnector extends Actor with ActorLogging with Instrumentat
         }
     }
 
+    /**
+      * Initialize the Jena Fuseki triplestore. Currently only works for 'knora-test'
+      * and 'knora-test-unit' repository names.
```

Comment:
Why?

Reply:
because here I'm using the two Fuseki configs that we have (knora-test and knora-test-unit). I guess I can make it configurable, but then maybe in another PR. I've created https://dasch.myjetbrains.com/youtrack/issue/DSP-434.

Reply:
I'm just trying to understand the use case for this method. Does this method only need to be called for `knora-test` and `knora-test-unit`, or is this a limitation that needs to be dealt with?

Reply:
This method, in my opinion, mainly makes sense for automated testing, to make life easier. To be used, the API needs to be started with a certain flag, which I personally would never use or recommend for production.

Reply:
OK, then please just clarify that in the comment.

Reply:
done